repos/spack_repo/builtin/packages/nccl_tests/package.py

```diff
@@ -16,9 +16,9 @@ class NcclTests(MakefilePackage, CudaPackage):
     homepage = "https://github.com/NVIDIA/nccl-tests"
     url = "https://github.com/NVIDIA/nccl-tests/archive/v2.0.0.tar.gz"
 
+    version("2.17.6", sha256="38974e70342150ede0e3fb8fd24bfe60ccbde90fa20241716ba8bb338e6b61a0")
     version("2.16.3", sha256="ec373552e216c0df03414718bd588585a1719971b20a5939a2f9488c8ce767eb")
     version("2.13.6", sha256="52b472a58a4918d3221a9b8c4bd9335382643d7e241983918b64692d685cc3d1")
-    version("2.0.0", sha256="731fc3b7c37de59cfe880bf198349ac185639ef23570749ea6aef334c850c49c")
```

Comment:
This tag still exists in the repo. Maybe it should be marked as deprecated before removing it?